pkg/storageos/rbac.go

```diff
@@ -264,7 +265,7 @@ func (s *Deployment) createClusterRoleForScheduler() error {
 		{
 			APIGroups: []string{"events.k8s.io"},
 			Resources: []string{"events"},
-			Verbs:     []string{"create"},
+			Verbs:     []string{"create", "patch"},
```

Comment:
This permission should also be granted to the operator first like for pod status permission.